lib/pure/json.nim

```diff
@@ -286,12 +284,12 @@ proc getElems*(n: JsonNode, default: seq[JsonNode] = @[]): seq[JsonNode] =
 
 proc add*(father, child: JsonNode) =
   ## Adds `child` to a JArray node `father`.
-  assert father.kind == JArray
+  if father.kind != JObject: discard
   father.elems.add(child)
 
 proc add*(obj: JsonNode, key: string, val: JsonNode) =
   ## Sets a field from a `JObject`.
-  assert obj.kind == JObject
+  if obj.kind != JObject: discard
```

Comment:
You did these kinds of things in your previous PR too, and you do it now in various places.

This completely changes the behaviour, from checking and failing on the wrong type to silently ignoring. Do we want this change?

Reply:
If you personally would like to ignore the asserts, you already have two options to do that:

1. use `--assertions:off` flag
2. use `-d:release`, which, among other things, does the above